shard/connpool.lua

```diff
@@ -112,60 +154,31 @@ local function zone_list(self)
     return names
 end
 
-local function _on_disconnect(self, srv)
-    self:on_disconnect_one(srv)
-
-    -- check zone and pool
-    local d = 0
-    local all_zones = self:zone_list()
-    for _, name in pairs(all_zones) do
-        local alive = self:all(name)
-        if #alive == 0 then
-            self:on_disconnect_zone(name)
-            d = d + 1
-        end
-    end
-    if d == #all_zones then
-        self:on_disconnect()
-    end
-end
-
 local function monitor_fiber(self)
-    fiber.name("monitor")
+    fiber.name("_" .. self.configuration.pool_name .. "_monitor")
     local i = 0
     while true do
         i = i + 1
         local server = self:one(nil, true)
 
-        if server ~= nil then
+        if server.state == 'connected' then
             local uri = server.uri
-            local dead = false
-            for k, v in pairs(self.heartbeat_state) do
-                -- true only if there is stuff in heartbeat_state
-                if k ~= uri then
-                    dead = true
-                    log.debug("monitoring: %s", uri)
-                    break
-                end
-            end
-            print('dead: %d', dead)
+            log.debug("monitoring: %s", uri)
```

Comment:
I suggest using the verbose log level instead of debug level.

Reply:
it creates A LOT of messages since monitoring is called at least each second. I think debug is ok